Source/CsDebugScript.Engine/Engine/Utility/SimpleCache.cs

```diff
@@ -91,13 +142,27 @@ public T Value
                 Cached = true;
             }
         }
+
+        /// <summary>
+        /// Explicit implementation of ICache interface in order not to bloat up the namespace.
+        /// Prefered way is to use generic Value.
+        /// </summary>
+        object ICache.ValueRaw { get { return value; } }
```

Comment:
Field removed.

---
In reply to: [70603077](https://github.com/southpolenator/WinDbgCs/pull/8#discussion_r70603077) [](ancestors = 70603077)